lib/topological_inventory/openshift/logging.rb

```diff
@@ -0,0 +1,17 @@
+module TopologicalInventory
+  module Openshift
+    class << self
+      attr_writer :logger
+    end
+
+    def self.logger
+      @logger ||= Logger.new(STDOUT, :level => Logger::INFO)
```

Comment:
I think this should use `ManageIQ::Loggers::Container` if possible.  https://github.com/ManageIQ/manageiq-loggers/blob/master/lib/manageiq/loggers/container.rb

Reply:
:+1: will do in a follow-up